src/app/config/config.py

```diff
@@ -183,6 +197,8 @@ def __init__(self):
         super().__init__()
         self.TEST_DATASET_PATH = get_value_from_env("TEST_DATASET_PATH")
         self.TEST_SCHEMA_PATH = get_value_from_env("TEST_SCHEMA_PATH")
+        self.SURVEY_MAP_URL = get_value_from_env("SURVEY_MAP_URL")
 
     TEST_DATASET_PATH: str
     TEST_SCHEMA_PATH: str
+    SURVEY_MAP_URL: str
```

Comment:
Sorry I have missed this. Can you remove survey_map_url from this class as it is within class config already

Reply:
No worries, just removed the duplicate one.